MsixCore/msixmgr/CommandLineInterface.cpp

```diff
@@ -23,6 +23,34 @@ std::map<std::wstring, Options, CaseInsensitiveLess> CommandLineInterface::s_opt
             commandLineInterface->m_operationType = OperationType::Add;
             commandLineInterface->m_packageFilePath = utf8_to_utf16(path);
             return S_OK;
+        },
+        {
+            {
+                L"-sourceApplicationId",
+                Option(true, IDS_STRING_HELP_OPTION_UNPACK_DESTINATION,
+                    [&](CommandLineInterface* commandLineInterface, const std::string& sourceApplicationId)
+                {
+                    if (commandLineInterface->m_operationType != OperationType::Add)

```

Comment:
I don't think this check is needed here as sourceApplicationId is present for all the parent arguments.

Reply:
Removed as suggested